packages/connect-react/src/components/ErrorBoundary.tsx

```
@@ -0,0 +1,26 @@
+
+type Props = {
+  children: React.ReactNode 
+  fallback: (err: any) => React.ReactNode 
+}
+
+export class ErrorBoundary extends Component<Props> {
+  state = {err: undefined}
+
+  constructor(props: Props) {
+    super(props)
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)